libraries/classes/EditField.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpMyAdmin;
+
+/**
+ * @psalm-immutable
+ */
+final class EditField
+{
+    /** @var string $columnName */
+    public $columnName;
+    /** @var string $value */
+    public $value;
+    /** @var string $type */
+    public $type;
+    /** @var bool $autoIncrement */
+    public $autoIncrement;
+    /** @var bool $isNull */
+    public $isNull;
+    /** @var bool $wasPreviouslyNull */
+    public $wasPreviouslyNull;
+    /** @var string $function */
+    public $function;
+    /** @var ?string $salt */
```

Comment:
```suggestion
    /** @var string|null $salt */
```
This is coherent with the codebase and is phpdoc standard syntax valid ;)

Reply:
Thanks. I will change this. This is only a very rough sketch of what I want to do. I still have a lot of work to do here.